Anna_Gamzinova/api_collections/test_api.py

```diff
@@ -0,0 +1,103 @@
+from http import HTTPStatus
+
+from Anna_Gamzinova.api_collections.people_collection import PeopleAPI
+
+
+def test_get_person_200():
+    response = PeopleAPI().get_person(300)
+    print(response.json())
+    assert response.status_code == HTTPStatus.OK, f'Status code is not as expected\n Actual: {response.status_code}' \
+                                                  f'\nExpected: {HTTPStatus.OK}'
+
+
+def test_get_people_200():
+    response = PeopleAPI().get_list_of_people()
+    assert response.status_code == HTTPStatus.OK, f'Status code is not as expected\n Actual: {response.status_code}' \
+                                                  f'\nExpected: {HTTPStatus.OK}'
+
+
+def test_create_person():
+    new_user = {
+        "name": "Santa Clause",
+        "gender": "male",
+        "email": "hohoho1@gmail.com",
+        "status": "active"}
```

Comment:
it is better to replace it with a data class.

Reply:
fixed